tests/orders/tests-edd-payment.php

```diff
@@ -549,7 +549,7 @@ public function test_modify_cart_item_price() {
 		$this->assertEquals( 1, $this->payment->cart_details[0]['item_price'] );
 
 		$download = new \EDD_Download( $this->payment->cart_details[0]['id'] );
-		$this->assertEquals( 41, $download->get_earnings() );
+		$this->assertEquals( 1.00, $download->get_earnings() );
```

Comment:
I did a lot of logging during tests and I think the changes here are correct--it seems like the original modifications were adding to the download earnings instead of changing. So my understanding is that this one payment goes through:
1. Starts with a `40.00` order item total so that's the download [net] earnings.
2. The cart item price is changed to `1.00` so the download [net] earnings should be `1.00`.
3. The cart item price is changed to `3.00` in the next test so the [net] earnings should be `3.00`.